FastSimulation/Tracking/python/MixedTripletStep_cff.py

```diff
@@ -0,0 +1,59 @@
+import FWCore.ParameterSet.Config as cms
+
+# import the full tracking equivalent of this file
+import RecoTracker.IterativeTracking.MixedTripletStep_cff
+
+# trajectory seeds
+import FastSimulation.Tracking.TrajectorySeedProducer_cfi
+mixedTripletStepSeeds = FastSimulation.Tracking.TrajectorySeedProducer_cfi.trajectorySeedProducer.clone(
+    simTrackSelection = FastSimulation.Tracking.TrajectorySeedProducer_cfi.trajectorySeedProducer.simTrackSelection.clone(
+        skipSimTrackIds = [
+            cms.InputTag("initialStepSimTrackIds"),
+            cms.InputTag("detachedTripletStepSimTrackIds"),
+            cms.InputTag("lowPtTripletStepSimTrackIds"),
+            cms.InputTag("pixelPairStepSimTrackIds")],
+        pTMin = 0.15,
+        maxD0 = 10.0,
+        maxZ0 = 30
+        ),
+    minLayersCrossed = 3,
+    originpTMin =  RecoTracker.IterativeTracking.MixedTripletStep_cff.mixedTripletStepSeedsA.RegionFactoryPSet.RegionPSet.ptMin,
+    originRadius = RecoTracker.IterativeTracking.MixedTripletStep_cff.mixedTripletStepSeedsA.RegionFactoryPSet.RegionPSet.originRadius,
+    originHalfLength = RecoTracker.IterativeTracking.MixedTripletStep_cff.mixedTripletStepSeedsA.RegionFactoryPSet.RegionPSet.originHalfLength,
+    layerList = RecoTracker.IterativeTracking.MixedTripletStep_cff.mixedTripletStepSeedLayersA.layerList.value()+RecoTracker.IterativeTracking.MixedTripletStep_cff.mixedTripletStepSeedLayersB.layerList.value()
+)
+# Fullsim has different cuts for A and B, here in FastSIm only A cuts are implemented 
+# need to do something about this ??
+
```

Comment:
@shchauha 

can you sync this iteration further with full tracking
full tracking defines 2 seed collections in this iteration
http://cmslxr.fnal.gov/lxr/source/RecoTracker/IterativeTracking/python/MixedTripletStep_cff.py

mixedTripletStepSeedsA
mixedTripletStepSeedsB

you could define these for fast tracking, and then import the seedcombiner
```
mixedTripletStepSeeds = RecoTracker.IterativeTracking.MixedTripletStep_cff.mixedTripletSeeds
```
in principle the seed combiner should work properly in FastSim